goldens/api/coverage.api.md

```diff
@@ -0,0 +1,38 @@
+## API Report File for "@betterer/coverage"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { BettererTest } from '@betterer/betterer';
+
+// @public
+export type BettererCoverageDiff = BettererCoverageIssues;
+
+// @public
+export type BettererCoverageIssue = Record<BettererCoverageTypes, number>;
+
+// @public
+export type BettererCoverageIssues = Record<string, BettererCoverageIssue>;
+
+// @public
+export type BettererCoverageTest = BettererTest<BettererCoverageIssues, BettererCoverageIssues, BettererCoverageDiff>;
+
+// @public
+export interface BettererCoverageTestOptions<TotalCoverage extends boolean = boolean> {
+    baseDir: string;
+    coverageFile: string;
+    excludeFiles: TotalCoverage extends true ? [] : Array<RegExp>;
+    includeFiles: TotalCoverage extends true ? [] : Array<RegExp>;
+    totalCoverage: TotalCoverage;
+}
+
+// @public
+export type BettererCoverageTypes = 'lines' | 'statements' | 'functions' | 'branches';
+
+// @public (undocumented)
```

Comment:
Would like to make sure this is documented like the other built-in tests too

Reply:
done